packages/block-editor/src/store/selectors.js

```diff
@@ -1936,9 +1936,26 @@ const buildBlockTypeItem =
 			blockType.name,
 			'inserter'
 		);
+
+		let initialAttributes = {};
+
+		const hookedBlocksForCurrentBlock = getBlockTypes()?.filter(
+			( { blockHooks } ) => blockHooks && id in blockHooks
+		);
+
+		if ( hookedBlocksForCurrentBlock?.length ) {
+			initialAttributes = {
+				metadata: {
+					ignoredHookedBlocks: hookedBlocksForCurrentBlock.map(
+						( { name } ) => name
+					),
+				},
+			};
+		}
+
 		return {
 			...blockItemBase,
-			initialAttributes: {},
+			initialAttributes,
```

Comment:
I was glad to learn that we already had an `initialAttributes` arg, as that's exactly what we'd like to set here.